webapp/channels/src/types/store/draft.ts

```diff
@@ -35,7 +35,7 @@ export type PostDraft = {
 
 export function isPostDraftEmpty(draft: PostDraft): boolean {
     const hasMessage = draft.message.trim() !== '';
-    const hasAttachment = draft.fileInfos?.length > 0 || draft.file_ids?.length;
+    const hasAttachment = draft.fileInfos?.length > 0;
```

Comment:
Note to reviewer-

The `draft.file_ids` check was added earlier when merging advanced text editor into master. Later on, I updated the selector that fetches the editing post, to include the file and all the metadata. So the `draft.file_ids` is not needed now.